File: ARCADE/The Core/Level 5_1: createArray.cs
    using System;
    using System.Collections.Generic;
     
    public class Test
    {
    	public static int[] createArray(int size){
            var myList = new List<int>();
            for (int i = 0; i < size; i++){
                myList.Add(1);
            }
            var a = myList.ToArray();
            return a;
        }
    	public static void Main()
    	{
    		int size = 4;
    		var result = createArray(size);
    		foreach(int i in result){
    			Console.Write(i + " ");
    		}
    	}
    }

File: ARCADE/The Core/Level 5_1: createArray.pas
program createArray;
Type
    ArrayInteger = Array of integer;
function createArray(size: integer): ArrayInteger;
var A : ArrayInteger;
    i : integer;
begin
    SetLength(A, size);
    for i := 0 to length(A) - 1 do
        A[i] := 1;
    
    createArray := A;
end;

var size, i : integer;
    A : ArrayInteger;
begin
	size := 4;
	A := createArray(size);
	for i:= 0 to length(A) - 1 do
	    write(A[i], ' ');
end.
